src/pages/workspaces/migration/BillingProjectParent.ts

```diff
@@ -1,21 +1,88 @@
 import _ from 'lodash/fp';
-import { ReactNode } from 'react';
-import { div, h } from 'react-hyperscript-helpers';
+import pluralize from 'pluralize';
+import { ReactNode, useState } from 'react';
+import { div, h, span } from 'react-hyperscript-helpers';
 import Collapse from 'src/components/Collapse';
+import { ButtonOutline } from 'src/components/common';
+import { Ajax } from 'src/libs/ajax';
 import colors from 'src/libs/colors';
-import { BillingProjectMigrationInfo } from 'src/pages/workspaces/migration/migration-utils';
+import { reportErrorAndRethrow } from 'src/libs/error';
+import * as Utils from 'src/libs/utils';
```

Comment:
the `cond` in `src/libs/utils` is now just a re-export of the definition in `@terra-ui-packages/core-utils`

```suggestion
import { cond, DEFAULT } from '@terra-ui-packages/core-utils';
```

Reply:
Done!